index.html

```diff
@@ -19,14 +19,164 @@
+</Section>
+
+<!-- __________________________________CARD2______________________________ -->
+<Section class="card-container">
+  <div class="card card1">
+    <img src="assets/CardPhoto2.svg" alt="ImgCard2">
+    <div class="cardWorks">
+        <h4>Tonic</h4>
+      <div>
+        <span class="word1">CANOPY</span>
+        <span class="dot"></span>
+        <span class="word2">Back End Dev</span>
+        <span class="dot"></span>
+        <span class="word3">2015</span>
+      </div>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+        <ul class="workContainer">
+          <li id="workContainerhtml">HTML</li>
+          <li id="workContainercss">CSS</li>
+          <li id="workContainerjs">Javascript</li>
+        </ul>
+      <div>
+          <button type="button" class="cardButton">See Project</button>
+        </div>
+      </div>
+    </div>
+  </div>
+</Section>
+<!-- __________________________________CARD3______________________________ -->
+<Section class="card-container">
+  <div class="card card1">
+    <img src="assets/CardPhoto3.svg" alt="ImgCard3">
+    <div class="cardWorks">
+        <h4>Tonic</h4>
+      <div>
+        <span class="word1">CANOPY</span>
+        <span class="dot"></span>
+        <span class="word2">Back End Dev</span>
+        <span class="dot"></span>
+        <span class="word3">2015</span>
+      </div>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+        <ul class="workContainer">
+          <li id="workContainerhtml">HTML</li>
+          <li id="workContainercss">CSS</li>
+          <li id="workContainerjs">Javascript</li>
+        </ul>
+        <div>
+          <button type="button" class="cardButton">See Project</button>
+        </div>
+      </div>
+    </div>
+  </div>
+</Section>
+<!-- __________________________________CARD4______________________________ -->
+<Section class="card-container">
+  <div class="card card1">
+    <img src="assets/CardPhoto4.svg" alt="ImgCard4">
+    <div class="cardWorks">
+        <h4>Tonic</h4>
+      <div>
+        <span class="word1">CANOPY</span>
+        <span class="dot"></span>
+        <span class="word2">Back End Dev</span>
+        <span class="dot"></span>
+        <span class="word3">2015</span>
+      </div>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+        <ul class="workContainer">
+          <li id="workContainerhtml">HTML</li>
+          <li id="workContainercss">CSS</li>
+          <li id="workContainerjs">Javascript</li>
+        </ul>
+        <div>
+          <button type="button" class="cardButton">See Project</button>
+        </div>
+      </div>
+    </div>
+  </div>
+</Section>
+
+<!-- ______________________________About my self___________________________ -->
+<section class="aboutMyself">
+  <div class="aboutMyself-container">
+    <h3>About Myself</h3>
```

Comment:
As suggested by your previous reviewer, please ensure that you stick to the design of the Figma template as it is the core requirement of this part of your learning. 📖 

Your header text "About Myself" still needs a slight adjustment. ♻️ 

- [ ] The easiest recommendation I would give is to try reducing the `width` of the `<h3>` element you used. And please remember to use responsive measure units like `%`, `em`, or `rem` so that the width becomes uniform across all screen sizes. 🖥️ 📱